core/src/test/scala/matryoshka/spec.scala

```diff
@@ -466,9 +474,11 @@ class FixplateSpecs extends Specification with ScalaCheck with ScalazMatchers {
       }
     }
 
-    def extractFactors: Coalgebra[Exp, Int] = x =>
+    // TODO: Why do we need to explicitly call the implicit here? Is it because
```

Comment:
Did you happen to try defining `extractFactors: Int => Exp[Int]` and then allowing the implicit to convert at the use site?